### ISSUE TITLE
Installation from PyPi doesn't pull in dateutil package

### ISSUE THREAD

**Issue by stewartadam** [Accelize/pycosio#1]
After a `pipenv install pycosio`:
```
>>> import pycosio
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/stewartadam/devel/virtualenvs/tmp-d0yDwSVb/lib/python3.7/site-packages/pycosio/__init__.py", line 25, in <module>
    from pycosio._core.functions_io import cos_open as open
  File "/Users/stewartadam/devel/virtualenvs/tmp-d0yDwSVb/lib/python3.7/site-packages/pycosio/_core/functions_io.py", line 7, in <module>
    from pycosio._core.storage_manager import get_instance
  File "/Users/stewartadam/devel/virtualenvs/tmp-d0yDwSVb/lib/python3.7/site-packages/pycosio/_core/storage_manager.py", line 7, in <module>
    from pycosio._core.io_raw import ObjectRawIOBase
  File "/Users/stewartadam/devel/virtualenvs/tmp-d0yDwSVb/lib/python3.7/site-packages/pycosio/_core/io_raw.py", line 10, in <module>
    from pycosio._core.io_system import SystemBase
  File "/Users/stewartadam/devel/virtualenvs/tmp-d0yDwSVb/lib/python3.7/site-packages/pycosio/_core/io_system.py", line 9, in <module>
    from dateutil.parser import parse
ModuleNotFoundError: No module named 'dateutil'
```

**Comment by xlz-jgoutin** [comment]
The module was present in the `setup.py`, but a missing comma broke the dependencies install.

This was fixed in cb5bb9a.

**Comment by xlz-jgoutin** [comment]
Pycosio was updated to 1.2.1 on PyPI to fix this issue.
